Sources/MapboxMaps/Style/Generated/Sources/GeojsonSource.swift

```diff
@@ -67,7 +67,7 @@ Example: `{"sum": ["+", ["get", "scalerank"]]}`.
 For more advanced use cases, in place of `operator`, you can use a custom reduce expression that references a special `["accumulated"]` value, e.g.:
 `{"sum": [["+", ["accumulated"], ["get", "sum"]], ["get", "scalerank"]]}` 
     */
-    public var clusterProperties: GeoJSON?
+    public var clusterProperties: [String: Expression]?
```

Comment:
How does this interface support `[String: [Expression]]`? See the doc comment last example above the variable declaration:
`{"sum": [["+", ["accumulated"], ["get", "sum"]], ["get", "scalerank"]]}`

Reply:
I'm unable to create `[["+", ["accumulated"], ["get", "sum"]], ["get", "scalerank"]]` using the expression initializers. Does anyone have an example of how to do so?

Reply:
The more I dig into this the more I feel like the correct type for `clusterProperties` should be `[String: [Expression.Element]]

Reply:
@ZiZasaurus for visibility.

Reply:
See #1571